Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.12.2-alpine3.19
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHOUNNBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` path should not end with a slash. It should be `WORKDIR /app` to correctly set the working directory.